src/actions/entries.js

```diff
@@ -259,7 +259,7 @@ export function createEmptyDraft(collection) {
   return (dispatch) => {
     const dataFields = {};
     collection.get('fields', List()).forEach((field) => {
-      dataFields[field.get('name')] = field.get('default', null);
+      dataFields[field.get('name')] = undefined;
```

Comment:
@MichaelRomani We still want to allow users to manually set a default here, just default it to `undefined` instead of `null`. The code would be `field.get('default')`, just removing `null` as the second param like @erquhart mentioned.